### PR TITLE
Processor Graphs - use RRD's last recorded values instead of averages

### PR DESCRIPTION
**Pull request by ottorei**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

Add an optional mode to draw graphs using RRD's recorded LAST-value instead of AVERAGE. Option can be enabled from the UI and is disabled by default.

last_values

### CODE REVIEW

**Comment by murrant** [comment]
You've got slightly the wrong approach here.  You would never want to change this globally.
It should be graph specific.  Each graph should have a best way to show it's data.

**Comment by ottorei** [comment]
> 
> 
> You've got slightly the wrong approach here. You would never want to change this globally.
> It should be graph specific. Each graph should have a best way to show it's data.
> 
> Like for example sometimes it makes sense to display average and max.

What kind of approach would you suggest? Separate options for different type of graphs?

**Comment by murrant** [comment]
If you want to keep using the generic_multi_line.inc.php base graph, make an option to change the consolidation function.

In the file where the graph is generated, set the option to change the CF to AVERAGE, MINIMUM, MAXIMUM, and LAST as desired.

File: includes/html/graphs/device/processor_separate.inc.php [hunk 1]
```diff
@@ -27,4 +27,10 @@
 
 $nototal = 1;
 
+if (Config::get('webui.graph_processor_last_values')) {
```
Comment (murrant) [inline includes/html/graphs/device/processor_separate.inc.php]: Getting warmer, remove the option and just set `$use_last_values = true;`

**Review by murrant — APPROVED** [review]

**Comment by murrant** [comment]
Hmm, after thinking about this a bit.  I am not sure what the point is.  For non-aggregated data (5 min per point) average == last.
For aggregated data, showing the last basically discards a lot of the data.  If you want to see max and min values for aggregation, it should be show in addition to the average.

**Pull request closed without merging**